examples/vfkit.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The vmnet-helper authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Example for using vmnet-client with vfkit.
+
+MAC_ADDRESS="92:c9:52:b7:6c:08"
+DISK_IMAGE="disk.img"
+CIDATA_ISO="cidata.iso"
+
+if [ ! -f "$DISK_IMAGE" ]; then
+    ./download.sh "$DISK_IMAGE"
+fi
+
+if [ ! -f "$CIDATA_ISO" ]; then
+    ./create-iso.sh "$MAC_ADDRESS" "$CIDATA_ISO"
+fi
+
+# Start vfkit and vmnet-helper connected via unix datagram sockets.
+/opt/vmnet-helper/bin/vmnet-client \
+    vfkit \
+    --memory=1024 \
+    --cpus=1 \
+    --bootloader=efi,variable-store=efi-variable-store,create \
+    "--device=usb-mass-storage,path=$CIDATA_ISO,readonly" \
```

Comment:
This could make use of https://github.com/crc-org/vfkit/commit/bfa70a7eeb273c5497140a60e8ff78b3e9d37ca3 but it is not in a released version yet.

Reply:
Nice, but network-config is a must to avoid issues with multiple interfaces and dns.

Reply:
Ah, did not pay attention you have an extra file we don’t support :)